recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class AndreasbuhrCppCoroConan(ConanFile):
+    name = "andreasbuhr-cppcoro"
+    description = "A library of C++ coroutine abstractions for the coroutines TS"
+    topics = ("conan", "cpp", "async", "coroutines")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/andreasbuhr/cppcoro"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "10",
+            "clang": "8",
+            "apple-clang": "10",
+        }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

(together with the following suggestion)